challenges/functions.js

```diff
@@ -1,24 +1,41 @@
 // ==== Callbacks ====  
 
+console.log("Callbacks Challenges 👇")
```

Comment:
Love these little spots in the console that specifically tell you what you're looking at. Just had to point that out.